app/src/main/java/cm/aptoide/pt/notification/ComingSoonNotificationManager.kt

```diff
@@ -5,10 +5,12 @@ import android.app.NotificationManager
 import android.content.Context
 import android.os.Build
 import androidx.work.*
+import cm.aptoide.pt.home.AppComingSoonPreferencesManager
 import rx.Completable
 import java.util.concurrent.TimeUnit
 
-class ComingSoonNotificationManager(private val context: Context) {
+class ComingSoonNotificationManager(private val context: Context,
```

Comment:
This is related to last PR but ideally this class should not have a context and depend on the instance of workmanager and notificationmanager.